src/utils/rolls/combat-roll.ts

```diff
@@ -49,13 +49,20 @@ export interface CombatRollResult {
 }
 
 const getOutput = (value: number, effect: number): string => {
-  let output = `${value}`;
-
-  if (value > 0) {
-    output = effect ? `_**${value}**_` : `_${value}_`;
+  switch (value) {
+    case 1:
+      return `<:CD_ONE:889203020510933044>`;
+    case 2:
+      return `<:CD_TWO:889203051599114281>`;
+    case 3:
+    case 4:
+      return `<:CD_BLK:889203068091113583>`;
+    case 5:
+    case 6:
+      return `<:CD_EFT:889203079981957151>`;
+    default:
+      return `${value}`;
```

Comment:
This is really cool! Just one quick question, will these need to be added to a server/channel in order for them to work? And if so should we add something to the README regarding that?

- [ ] Review

Reply:
So, all discord bots essentially have nitro priviledges as far as emoji are concerned, so as long as the emoji exist it can happily post them anywhere. However, those are coded to ones that exist in a server I control, and if that server is deleted I'm not sure if they will cease functioning.

Reply:
Hmm interesting. I wonder if we should then drop the image files somewhere then as a backup.

Reply:
That can be done, not sure if we would hit any interesting copyright bits, but we could put a clear disclamer as to where the images are from.

Reply:
I agree. Credit to Modiphius should be added somewhere too since we do use some of the rule text.

Reply:
I've added a copy of the images and added a bit of a disclaimer with them, not sure if we might need to do more or not.

Reply:
I created #4 to better handle crediting and disclaimers. So I think that is enough for now.